lumicks/pylake/correlated_stack.py

```diff
@@ -115,6 +115,26 @@ def crop_by_pixels(self, x_min, x_max, y_min, y_max):
         data = self.src.with_roi(np.array([x_min, x_max, y_min, y_max]))
         return self.from_dataset(data, self.name, self.start_idx, self.stop_idx)
 
+    def get_image(self, channel=None):
+        """Get image data for the full stack as an `np.ndarray`.
+
+        Parameters
+        ----------
+        channel : {'red', 'green', 'blue', 'rgb', `None`}
+            The color channel of the requested data.
+            For single-color data, this argument is ignored.
+            For RGB data the default value `None` corresponds to 'rgb'.
```

Comment:
Can't we just make the default `"rgb"` and ignore the channel argument when it's a single color stack?

It doesn't seem like you have error handling for requesting `rgb` when there is no `rgb` anyhow?

Then you also don't need the `None` entry anymore in the `_cmap.`